src/plugins/dataflow-tool/nodes/controls/plot-button-control.scss

```diff
@@ -0,0 +1,25 @@
+.node-graph-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  color: white;
+
+  .graph-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    padding: 3px;
+    box-sizing: border-box;
+    border-radius: 1px;
+    border: 2px solid white;
+    background-color: white;
+
+    svg {
+      width: 14px;
+      height: 14px;
+      z-index: 1;
+    }
+  }
+}
```

Comment:
We generally try to avoid the lack of trailing newline (in part to avoid the 🚫 in the GitHub listing). I think there's a VSCode setting for adding it automatically on save.